src/components/Card/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Card.scss';
+
+const Card = ({ value, selected }) => (
+  <div className={`card ${selected ? 'selected' : ''}`}>
```

Comment:
Might be good to go ahead and pull in [`classnames`](https://github.com/JedWatson/classnames)